terraform/datadog/aws.tf

```diff
@@ -58,3 +58,34 @@ resource "datadog_monitor" "rds-failure" {
 
   tags = ["deployment:${var.env}", "service:${var.env}_monitors", "job:all"]
 }
+
+resource "datadog_monitor" "ec2-cpu-utilisation-warning" {
+  name                = "${format("%s CPU utilisation has been moderately high for a long time on {{bosh-job.name}}/{{bosh-index.name}}", var.env)}"
```

Comment:
Do you think you could change the name not to contain the variables. I don't think they work properly on DD. This maybe because of the formatter we use...

Reply:
😞 well spotted

Reply:
hmmm they work in the "events" page .... but not in the "triggered monitors" page (since the monitors page lists the monitors themselves, rather than the alerts/events generated from them).

It's weird, but I think it i _is_ correct?

Reply:
On the `events` page, do we perhaps care more for the `message` as oppose to the name of the monitor? I'm wondering in terms of consistency with others. 🤔 

![screen shot 2017-11-09 at 11 06 54](https://user-images.githubusercontent.com/2418945/32602510-30822162-c53e-11e7-828a-5106af92f4d1.png)